### PR TITLE
label music fade timer reset function

### PR DESCRIPTION
**Pull request by zelosos**

### CODE REVIEW

File: src/code/bank0.asm [hunk 1]
```diff
@@ -4190,9 +4190,11 @@ label_27DD::
     pop  bc                                       ; $27E6: $C1
     jp   ReloadSavedBank                          ; $27E7: $C3 $1D $08
 
-label_27EA::
-    ld   a, $38                                   ; $27EA: $3E $38
+resetMusicFadeTimer::
```
Comment (kemenaran) [inline src/code/bank0.asm:4193]: Nitpick: if this is a function, it should have a Capitalized name.

```suggestion
ResetMusicFadeTimer::
```

**Review by kemenaran — APPROVED** [review]
Looks good to me :)

File: src/code/bank0.asm (hunk 1 above)
Comment (zelosos) [reply src/code/bank0.asm]: need to be changed at all places. Could you do it please? I am working on a different branche at this moment

Comment (kemenaran) [reply src/code/bank0.asm]: Sure, no problem

Comment (kemenaran) [reply src/code/bank0.asm]: @zelosos I made the changes. But to update the PR, I need you to allow me to update your branch.

Can you check the ["Allow edits from maintainers"](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) checkbox on this PR?

Comment (zelosos) [reply src/code/bank0.asm]: was and is enabled as allways

Comment (zelosos) [reply src/code/bank0.asm]: I commited your suggestion. maybe this is the root cause

Comment (kemenaran) [reply src/code/bank0.asm]: Oh indeed, that was my fault (a mismatch in the branch name).

**Pull request merged**